plugins/azure/virtualmachinescaleset/scaleSetLinuxSSHEnabled.js

```diff
@@ -0,0 +1,56 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'VM Scale Set Linux SSH Enabled',
+    category: 'Virtual Machine Scale Set',
+    domain: 'Compute',
+    description: 'Ensures that Azure Virtual Machine scale sets with Linux OS has SSH enabled.',
+    more_info: 'SSH is an encrypted connection protocol that allows secure sign-ins over unsecured connections. SSH is the default connection protocol for Linux VMs hosted in Azure.',
+    link: 'https://learn.microsoft.com/en-us/azure/virtual-machines/linux/ssh-from-windows',
```

Comment:
is this correct? i think it should be this https://learn.microsoft.com/en-us/azure/virtual-machines/linux/mac-create-ssh-keys